CMakeLists.txt

```diff
@@ -738,6 +740,11 @@ endif()
 set(TARANTOOL_BUILD "${CMAKE_SYSTEM_NAME}-${CMAKE_SYSTEM_PROCESSOR}-${CMAKE_BUILD_TYPE}")
 set(TARANTOOL_C_COMPILER ${CMAKE_C_COMPILER})
 set(TARANTOOL_CXX_COMPILER ${CMAKE_CXX_COMPILER})
+if(ABORT_ON_LEAK)
+    set(TARANTOOL_OPTIONS "${TARANTOOL_OPTIONS} -DABORT_ON_LEAK=TRUE")
+else()
+    set(TARANTOOL_OPTIONS "${TARANTOOL_OPTIONS} -DABORT_ON_LEAK=FALSE")
+endif()
```

Comment:
Not sure if we want to export it to `tarantool.build.options`. Why do you do that?